translations/Arabic/special-terms.md

```diff
@@ -0,0 +1,48 @@
+# Terms
+
+- Message: رسالة
+  لقد وصلتك رسالة
+- Private Message (PM): رسالة خاصة
+  أرسل رسالة خاصة
+- Group PM: رسالة خاصة لمجموعة
+- Realm: نطاق
+  Closest translation possible
+- Stream: بث
+  Direct translation would be something like river stream. بث (which means something like TV broadcast) is the best translation in the context
+  شاهد البث المباشر
+- Topic: موضوع
+- Private/Invite-only Stream: بث خاص\بدعوة
+  يجب أن يكون لديك دعوة لتنضم لهذا البث الخاص
+- Public Stream: بث عام
+- Bot: روبوت
+  There is no translation for this word, so the Arabic writing for 'Robot' is used
+- Integration: تكامل
+- Notification: إشعار
+  وصلك إشعار جديد
+- Alert Word: كلمة تنبيهية
+- View: شاهد
+- Home: المنزل
+- Emoji: إيموجي
+  Even though the correct translation would be رموز تعبيرية, it is known simply as 'emoji', and so the Arabic writing for 'emoji' is used
+ 
+ 
+# Phrases
+
+- Subscribe to a stream/Unsubscribe from a stream: تابع هذا البث/إلغاء المتابعة
```

Comment:
I think it is understood here from context. All Arabic versions of famous apps (like Youtube) use it this way (like "subscribe to stream" / "cancel your subscription")